pages/cart_page/cart.css

```diff
@@ -0,0 +1,58 @@
+.cart-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
If you wanted to achieve responsiveness, add a media query. This is will look too small on mobile screens

Reply:
it is not yet made for mobile screens...i will make it responsive later on while refactoring